packages/RawButton/src/RawButton.js

```diff
@@ -86,7 +86,7 @@ const RawButton = React.forwardRef((props, ref) => {
       onKeyDown={handleKeyDown}
       onKeyUp={handleKeyUp}
       ref={rawButtonRef}
-      tabIndex={isDisabled ? -1 : tabIndex}
+      tabIndex={isDisabled ? null : tabIndex}
```

Comment:
shouldn't this be undefined so the tabIndex don't even render on the component?

Reply:
We tested it and it doesn't add the attribute with `null` either! Maybe this is a difference between React 15 and 16?

Reply:
It won't be rendered when using `null` as well